resources/js/components/studentProfile/CurrentInterests.vue

```diff
@@ -0,0 +1,12 @@
+<template>
+	<div class="row">
```

Comment:
You don't need this div or the following one. Grid stuff has already been handled in the parent.